include/GafferScene/InteractiveRender.h

```diff
@@ -37,11 +37,12 @@
 #ifndef GAFFERSCENE_INTERACTIVERENDER_H
 #define GAFFERSCENE_INTERACTIVERENDER_H
 
+#include "Gaffer/TypedObjectPlug.h"
```

Comment:
Move to Gaffer section below GafferScene.

Reply:
https://github.com/GafferHQ/gaffer/pull/3790/commits/04ec13379be81f71ca9bdf34063e38ef9861a24e